libcontainer/internal/userns/userns_maps_linux.c

```diff
@@ -1,3 +1,5 @@
+//go:build linux
```

Comment:
Do we need both of the comment line and the file name suffix?

Reply:
idk but that seems to be the convention.

Reply:
It's a tad redundant but I guess OK to have just in case.